src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "./App";
+
+test("test renders TK Recipe App", () => {
+  const { getByText } = render(<App />);
```

Comment:
we usually import `screen` from `@testing-library/react` and we use it as

`screen.getByText(...`

this way you don't have to extract all the methods from the render call